libs/wallet/src/web3-react/updater.ts

```diff
@@ -74,7 +74,7 @@ function _useSafeInfo(walletInfo: WalletInfo): GnosisSafeInfo | undefined {
           })
         )
         .catch((error) => {
-          console.error('[WalletUpdater] Error getting Safe Info', error)
+          console.info('[WalletUpdater] Error getting Safe Info', error)
```

Comment:
Change to info as now this is checked with every wallet, not only when it's a Safe, and creates a lot of noise.

Reply:
Maybe even debug. And u could change the message `address XXXX is not a safe (APi didn't return safe info)` 

at least add something like that as comment, cause next person can forget about this, and change info back to error